include/megaapi.h

```diff
@@ -5172,6 +5183,15 @@ class MegaSync
         NEW_TEMP_DISABLED = 6, // new sync added as temporarily disabled due to a temporary error
     };
 
+    enum SyncType
+    {
+        TYPE_UNKNOWN = 0x00,
+        TYPE_UP = 0x01, // sync up from local to remote
+        TYPE_DOWN = 0x02, // sync down from remote to local
```

Comment:
Well, we could comment them out, but then we would need to hardcode the value for `TWO_WAY`.
And in practice, the official apps won't start using those types randomly. Also, the `MegaSync` objects seen by the app are always provided by the SDK, so none of them will have `UP` or `DOWN`, since there's no MegaApi method to create such config :) 
The only inconvenient of keeping UP and DOWN here is... the fake impression that we support one way sync, while we don't. I don't see it a problem.

Reply:
Is there anything to change after this discussion?

Reply:
Ok let's leave it as is for now, thanks